presto-druid/src/main/java/com/facebook/presto/druid/DruidFilterExpressionConverter.java

```diff
@@ -184,7 +184,7 @@ public DruidExpression visitCall(CallExpression call, Function<VariableReference
             if (!(argument instanceof ConstantExpression)) {
                 throw new PrestoException(DRUID_PUSHDOWN_UNSUPPORTED_EXPRESSION, "Invalid Timestamp literal in Druid filter: " + argument.toString());
             }
-            SqlTimestamp value = new SqlTimestamp((long) ((ConstantExpression) argument).getValue(), session.getSqlFunctionProperties().getTimeZoneKey());
+            SqlTimestamp value = new SqlTimestamp((long) ((ConstantExpression) argument).getValue());
```

Comment:
I don't think we can ignore the timezone here.  The session here is the ConnectorSession,  we cannot assume the connection to druid is always using UTC (though now it's just using UTC).

>  By default, time operations use the UTC time zone. You can change the time zone by setting the connection context parameter "sqlTimeZone" to the name of another time zone, like "America/Los_Angeles", or to an offset like "-08:00". If you need to mix multiple time zones in the same query, or if you need to use a time zone other than the connection time zone, some functions also accept time zones as parameters. These parameters always take precedence over the connection time zone.
from Druid https://druid.apache.org/docs/latest/querying/sql.html#time-functions

Reply:
Hi,
The original code here is definitely a bug here and the result is not consistent with other connector like Hive. For example, the client is in Asia/Shanghai. If I do not change the code:
presto> select now();
```
                 _col0                 
---------------------------------------
 2020-08-07 13:11:15.704 Asia/Shanghai 
(1 row)
```

```
presto> SELECT __time FROM druid.druid.wikipedia where __time > timestamp '2015-09-12 00:00:00.000' and __time < timestamp '2015-09-12 01:00:00.000' limit 10;
         __time          
-------------------------
 2015-09-12 08:46:58.771 
 2015-09-12 08:47:00.496 
 2015-09-12 08:47:05.474 
 2015-09-12 08:47:08.770 
 2015-09-12 08:47:11.862 
 2015-09-12 08:47:13.987 
 2015-09-12 08:47:17.009 
 2015-09-12 08:47:19.591 
 2015-09-12 08:47:21.578 
 2015-09-12 08:47:25.821 
(10 rows)

Query 20200807_040053_00004_xsw2e, FINISHED, 1 node
Splits: 17 total, 17 done (100.00%)
0:02 [10 rows, 90B] [4 rows/s, 43B/s]
```
Is it reasonable? The client will convert the result time to  Asia/Shanghai again. This behavior is not consistent with other connectors, like hive.

Reply:
I agree with you, it's definitely a bug,  but I think the root cause is we are not passing the timezone into druid by the context parameter "sqlTimeZone".    

According to the doc, the default timezone in Druid is UTC, so we can just use the millisecond value in UTC as you said ((ConstantExpression) argument).getValue() is already in UTC.  For sure, your change could fix the issue, but we might lost the flexibility of setting the sqlTimeZone on client side.   

Just for now, I'm fine with either way, your call.